### ISSUE TITLE
EE polluting BB ajax responses without compatibility shim

### ISSUE THREAD

**Issue by mnelson4** [eventespresso/eea-beaver-builder#17]
BB 2.1 contains this code

```
/**
* Fixes ajax issues with Event Espresso plugin when builder is open.
* @since 2.1
*/
function fl_ee_suppress_notices() {
   if ( FLBuilderModel::is_builder_active() ) {
       add_filter( 'FHEE__EE_Front_Controller__display_errors', '__return_false' );
   }
}
add_action( 'wp', 'fl_ee_suppress_notices' );

/**
* Dont load ee core, fixes the text editor tabs missing issue.
* @since 2.1
*/
function fl_ee_no_load_builder_active() {
   if ( isset( $_GET['fl_builder'] ) && class_exists( 'EE_System' ) ) {
       remove_action( 'init', array( EE_System::instance(), 'core_loaded_and_ready' ), 9 );
   }
}
add_action( 'after_setup_theme', 'fl_ee_no_load_builder_active' );

/**
* Stops ee from outputting HTML into our ajax responses.
* @since 2.1
*/
function fl_ee_before_ajax() {
   add_filter( 'FHEE__EE_Front_Controller__display_errors', '__return_false' );
}
add_action( 'fl_ajax_before_call_action', 'fl_ee_before_ajax' );
```

On #15 Simon Prosser from BB agreed `fl_ee_no_load_builder_active` is no longer necessary, but I'd like to investigate the other compatibility shims and see if those might be affecting other plugins and whether EE core should make any adjustments

**Comment by mnelson4** [comment]
oh also, Simon said they were getting those AJAX errors anytime a BB module was saved (when those lines weren't in there)

**Comment by mnelson4** [comment]
The full version of BB that I have doesn't work with the latest version of WP, so I need an updated version. (The lite version works, but it also doesn't have the code in question). @sethshoultes could you send me it if you have it? Thanks

**Comment by mnelson4** [comment]
I tried removing those lines from the BB plugin and am not reproducing seeing any errors in the BuddyPress ajax responses when building and saving a page.
I'm going to see if @Pross can give any more tips on how to reproduce this...

**Comment by mnelson4** [comment]
@Pross and I were unable to reproduce this issue.
Here's what we did:
-commented out `fl_ee_suppress_notices()` and `fl_ee_before_ajax()` from BB, activated EE, and created a page using Beaver Builder. No errors in ajax, everything worked fine
-I even purposefully added an EE error on every page by editing `ee/core/EE_System.php`'s method `initialize()` by adding `EE_Error::add_error('error on every page', __FILE__, __FUNCTION__, __LINE__);`, and then tried using Beaver Builder. I saw EE's error message when opening the page in Beaver Builder (which is expected), but no ajax requests were polluted.

So, this issue was probably fixed at some point, but I'm not certain. Besides, it might be good to leave those lines in BB in case someone's using an older version of EE that caused this problem. Besides, those lines aren't causing any problems.

So @Pross will probably just leave those lines in BB for now. But they're possibly unnecessary now
